auth-server/migrations/20200111182158_email_undo_token.js

```diff
@@ -0,0 +1,16 @@
+// Table to store tokens for email verification
+exports.up = function(knex) {
+	return knex.schema.createTable('email_undo_token', (table) => {
+		table.increments('id').primary().notNullable();
+		table.string('token').unique().notNullable();
```

Comment:
There's a buit in `.uuid` for knex and postgres that we should use here (and where we have tokens actually).

Reply:
done